src/main/resources/swagger.api/schemas/dto/tenant.json

```diff
@@ -4,17 +4,17 @@
   "Tenant": {
     "type": "object",
     "properties": {
-      "tenantId": {
+      "id": {
```

Comment:
Based on required contract we have tenantId and tenantName:
![image](https://user-images.githubusercontent.com/25097693/224737102-ef81c51a-97bc-4ca9-af65-6d754e8c7a65.png)

Reply:
Olamide left commit about it. he agrees with this style